src/braket/circuits/noise_helpers.py

```diff
@@ -0,0 +1,220 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Iterable
+
+from braket.circuits.instruction import Instruction
+from braket.circuits.moments import Moments
+from braket.circuits.noise import Noise
+from braket.circuits.qubit_set import QubitSet, QubitSetInput
+
+if TYPE_CHECKING:  # pragma: no cover
+    from braket.circuits.circuit import Circuit
+
+
+def _add_noise(
+    circ: Circuit,
+    noise: Noise,
+    target_gates: Iterable[str],
```

Comment:
These again should be Optional

Reply:
They are actually not optional for this function. Default values are passed to this function if users don't specify.